documentation/modules/vmware-prerequisites.adoc

```diff
@@ -5,11 +5,11 @@
 [id="vmware-prerequisites_{context}"]
 = VMware prerequisites
 
-It is strongly recommended to create a VDDK image to accelerate migrations. For more information, see xref:../master.adoc#creating-vddk-image_mtv[Creating a VDDK image].
+It is strongly recommended to create a VDDK image to accelerate migrations. For more information, see xref:../master.adoc#creating-vddk-image_vmware[Creating a VDDK image].
```

Comment:
Is it enough to begin with, "Create..." since this is a pre-req?  Or, why is it strongly recommended?

Reply:
Rewritten as "Tip"